datashader/core.py

```diff
@@ -3,19 +3,36 @@
 from numbers import Number
 from math import log10
 import warnings
+from importlib.util import find_spec
 
 import numpy as np
 import pandas as pd
-import dask.dataframe as dd
+import dask
 import dask.array as da
 from xarray import DataArray, Dataset
 
+query_planning = dask.config.get("dataframe.query-planning")
+dask.config.set({"dataframe.query-planning": False})
```

Comment:
Does datashader just not work with query planning? It seems a little rude to disable it entirely on import.